CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.2.1
```

Comment:
Since this is new functionality shouldn't it be 1.3.0?

Reply:
Corrected